maps/site53/job/jobs/command.dm

```diff
@@ -164,37 +164,108 @@ ut // COMMAND
 
 // MISC
 
-/datum/job/o5rep
-	title = "O5 Representative"
-	department = "Command"
+/datum/job/tribunal
+
+	title = "Internal Tribunal Department Officer"
+	department = "Civilian"
+	selection_color = "#2f2f7f"
 	department_flag = COM
 	total_positions = 1
 	spawn_positions = 1
-//	//duties = "<big><b>As the GOC Representative, your task is to assess the facility and generally advocate for hardline approaches in regards to anomalies and their containment, or destruction. You value human lives far over any anomaly, as does the Global Occult Coalition, and should see to it that lives are preserved where possible, even D-Class ones. Though combat is not your duty, you are issued a revolver to defend yourself with. This job is heavy roleplay: you're expected to be well-versed in actually talking to people on the matters described. Containment of SCPs and direct site matters are not your matters, so don't get involved.</b></big>"
-//	//supervisors = "Global Occult Coalition Regional Command"
-	economic_power = 5
+	supervisors = "The Tribunal Department"
+	economic_power = 4
 	minimal_player_age = 5
-	minimal_player_age = 9
 	ideal_character_age = 30
-	alt_titles = list("Ethics Committee Representative")
```

Comment:
you should leave this alt title for the ethics committee liaison and give the tribunal officer another one imho

Reply:
I’m afraid I don’t understand, the roles have been split in two, no need for an alt title.